README.md

```diff
@@ -661,183 +661,40 @@ correctness, by implementing GC synchronously, as part of the delete request. To
 the best of my knowledge, there's nothing in the description that _requires_ GC
 to occur asynchronously.
 
-#### Delete step 2a
-
-> Replicates an empty value to all nodes by executing the identity transform
-> with 2F+1 quorum size. Reschedules itself if at least one node is down.
-
-Here we have another application of the now-familiar gambit of performing an
-identity read, presumably to increment ballot numbers beyond the delete
-operation. What's interesting (and in some sense quite amusing) is that we need
-to do this against every node (meaning proposer) and should retry the whole
-operation if any node (proposer) is down. That's a very brute-force approach to
-GC, which is nice and practical — and probably amenable to batching.
-
-```go
-func gcBroadcastIdentity(key string, proposers []Proposer) error {
-    identity := func(x State) State { return x }
-    for _, p := range proposers {
-        if _, err := p.Propose(key, identity); err != nil {
-            return err
-        }
-    }
-    return nil
-}
-```
-
-#### Delete steps 2b, 2c
-
-> For each proposer, fast-forwards its counter to generate ballot numbers
-> greater than the tombstone's number.
->
-> Wait some time to make sure that all in-channel messages with lesser ballots
-> were delivered.
-
-This is quite interesting, and somewhat vague: what's a tombstone? What channel
-are we talking about? If we read further below, we get some hints.
-
-> The steps 2b and 2c are necessary to eliminate a situation when a register was
-> removed from F+1 acceptors, and they accepted [a] value with [a] lesser ballot
-> number than the ballot number of the empty value still stored on [the] other F
-> acceptors.
+One note from below:
 
-This implies that tombstone is the ballot number (or, more specifically, the
-ballot number counter) of the identity read transform we've just executed. But
-we've presumably executed that transform via the normal proposer.Propose method,
-which doesn't return anything about its ballot number! So, assuming we don't
-want to create a new proposal mechanism for the delete process, it seems we'll
-need to make a modification to the Propose method, to return the resulting
-ballot number.
+> Each step of the GC process is idempotent so if any acceptor or proposer is
+> down the process reschedules itself.
```

Comment:
More generally, the GC process always runs, whether or not there are tombstones to reap?